src/app/store/step/step.store.ts

```diff
@@ -192,4 +192,10 @@ export class StepStore {
   getSteps() {
     return this.steps;
   }
+
+  // Check if we need a custom form handling which stores the parsed
+  // properties in customProperties
+  isCustomStep(step: Step) : boolean {
+    return step.stepKind === BASIC_FILTER;
```

Comment:
If we could agree on a custom handling that would be awesome. However I found that the handling was a bit different, e.g. when the datamapper step finishes, a different navigation [happens](https://github.com/rhuss/syndesis-ui/blob/4d0760d8ad5e36845616bff93ed101a17cde1128/src/app/integrations/edit-page/step-configure/step-configure.component.ts#L68)

For https://github.com/rhuss/syndesis-ui/blob/4d0760d8ad5e36845616bff93ed101a17cde1128/src/app/integrations/edit-page/step-configure/step-configure.component.ts#L145 I think we can combine both easily.